parm/atm/jcb-base.yaml.j2

```diff
@@ -27,6 +27,11 @@ final_prints_frequency: PT3H
 number_of_outer_loops: 2
 analysis_variables: [ua,va,t,ps,sphum,ice_wat,liq_wat,o3mr]
 
+# Testing
```

Comment:
Since jcb-algorithms will default `do_testing` to False itself I think you can move this key into your test/*/global-workflow/jcb-*.yaml files instead. This way we won't see anything about testing above the testing directory.

Reply:
@danholdaway Let's discuss tag up and discuss in-person